examples/nidmm_measurement/measurement.py

```diff
@@ -39,7 +61,7 @@
     instrument_type=nims.session_management.INSTRUMENT_TYPE_NI_DMM,
 )
 @measurement_service.configuration(
-    "measurement_type", nims.DataType.Enum, nidmm.Function.DC_VOLTS, enum_type=nidmm.Function
+    "measurement_type", nims.DataType.Enum, Function.DC_VOLTS, enum_type=Function
```

Comment:
This wrapper enum is necessary due to the requirement that all enums must have a zero value to be used with this enum feature. I don't remember exactly why this decision was made, but I think it was related to enums in the .proto file requiring a zero value. @synesthetize , do you have more context on why we made that design choice?

I originally tried to handle this internally and inject a zero value into the enum behind the scenes, but ran into an issue where generating annotations with the added zero would then it would be visible in the measurement panel. Additionally, a user might have an enum already named `NONE`/`UNKNOWN`/`INVALID`/etc. so whatever we name the zero value would be problematic if visible. I'm sure we could better handle enums without zero values if we added some more information in annotations to tell and work around it, but since not allowing enums with zero values was an intentional decision, I opted for just wrapping the enums instead.